src/components/leaderboard/TopThreeCard/index.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @next/next/no-img-element */
+import styles from '@/components/leaderboard/TopThreeCard/style.module.scss';
+
+interface UserCardProps {
+  position: 1 | 2 | 3;
+  rank: string;
+  name: string;
+  points: number;
+  image: string;
+}
+const positionNames = ['first', 'second', 'third'];
+
+const TopUserCard = ({ position, rank, name, points, image }: UserCardProps) => {
+  const trimName = name.length >= 25 ? `${name.substring(0, 24)}...` : name;
+  return (
+    <div className={styles.leaderboardCard} data-position={positionNames[position - 1]}>
+      <div className={styles.cardLeft}>{position}</div>
+      <div className={styles.cardRight}>
+        {
+          // eslint-disable-next- @next/next/no-img-element
+        }
+        <img className={styles.profileImage} src={image} alt="User Profile Pic" />
```

Comment:
We should actually be using the Next.js Image tag to help with performance everywhere we can which applies here.

Once we have the import from here.
```JS
import Image from 'next/image';
```

the only difference is that the size of the image has to be defined explicitly in pixels like
```HTML
<Image className={styles.profileImage} src={image} alt="User Profile Pic" width={80} height={80} />
```
Try not to disable errors you're getting unless it's really difficult to fix - most of the rules we have in place are there for good reason